src/printer.ts

```diff
@@ -100,15 +100,11 @@ export class PugPrinter {
 	private result: string = '';
 
 	private currentIndex: number = 0;
+	private currentLineLength: number = 0;
```

Comment:
@Shinigami92 I've moved this up here along with `currentIndex`, and I removed the JSDOC comment which wasn't actually meaningful for docs, and didn't say more then the code already does. And I've noticed one more thing: We are determining the length, not the actual position in a column, so this has to start at 0, meaning no content.